Automattic/woocommerce-payments
# Update colour coding of DisputeStatusChip based on dispute status and urgency

Fixes #6722 

#### Changes proposed in this Pull Request









This PR changes the colour of the Dispute Status Chip based on the following criteria:

* red = needs response within 72h
* yellow = needs response > 72h
* green = won
* blue = in progress / under review
* grey = lost


This Dispute Status Chip is rendered in the following locations, each with updated colours:

**Disputes List**
<img width="300" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/5a58f236-d00a-4477-a3ae-0b40166abe67"  />

**Dispute Details**
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/0d6b0cf4-be08-46fc-bb9e-85d9335695a2">

**Transaction Details**
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/120f3fa1-ec41-4333-9d49-f845856a9ee2">



#### Testing instructions





* Ensure your store has multiple disputes in various states, e.g. Lost, Won, Needs Response.
	* Possible statuses: `warning_needs_response`, `warning_under_review`, `warning_closed`, `needs_response`, `under_review`, `charge_refunded`, `won`, `lost`
	* Submit dispute evidence with `winning_evidence` in the `Additional Details` field to create a `Won` dispute. [See wiki](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows#winning-dispute).
* Navigate to Payments → Disputes. Ensure the colour of the chip under the **Status** column match the criteria listed above.
* Click on a dispute to view the Dispute Details screen. Ensure the status chip colour matches the criteria listed above.
* Click on the charge to view the Transaction Details screen. Ensure the status chip colour matches the criteria listed above.

-------------------


**Post merge**
